lib/src/js_proxies/js_object_map_proxy.dart

```diff
@@ -28,7 +28,7 @@ class JsObjectMapProxy<V> with MapMixin<String, V> {
   }
 
   @override
-  V remove(Object key) {
+  V? remove(Object? key) {
```

Comment:
Same thing as the above indexer comment, `key` should not be nullable or an `ArgumentError` should be explicitly thrown when it's null.

Reply:
this is not unwrapped to the non-nullable type by force, as it is propagated as is to the `get by index` operator. and as with `get by index`, `MapMixin` declares both nullable key and value here